package.json

```diff
@@ -22,6 +24,6 @@
     "app-module-path": "^2.x"
   },
   "devDependencies": {
-    "nodemon": "^1.17.1"
+    "localtunnel": "^1.8.3",
```

Comment:
Trailing comma here also seems to be an issue for me

Reply:
actually, why has nodemon gone? needs to stay in

Reply:
Great catch, I think that's a merge conflict on my end, will put it back in 👍